inductiva/resources/resources.py

```diff
@@ -0,0 +1,62 @@
+"""Functions to manage or retrieve user resources."""
+from absl import logging
+import inductiva
+from inductiva.client.apis.tags import instance_api
+from inductiva import resources
+
+
+def get_storage_size():
+    """Returns the total storage size in GB."""
```

Comment:
Does this mean the total storage size I have available or that I consumed?

Reply:
Consumed. I will rephrase it.